src/View/Helper/FormHelper.php

```diff
@@ -1008,7 +1008,8 @@ public function input($fieldName, array $options = [])
             'required' => null,
             'options' => null,
             'templates' => [],
-            'templateVars' => []
+            'templateVars' => [],
+            'optionsLabel' => true
```

Comment:
you are setting options so it should be `*options` ... and you are setting options for labels... so it should be labelsOptions or labelOptions... imho.